lib/common.py

```diff
@@ -75,6 +76,20 @@ def safe_split(text, sep=','):
     """ A split function that returns an empty list if the input string is empty. """
     return text.split(sep) if len(text) > 0 else []
 
+def strip_prefix(string, prefix):
+    if sys.version_info >= (3, 9):
+        return string.removeprefix(prefix)
```

Comment:
If happens that it was the code I initially wrote, before realizing that it would not work on 3.6.
Now we're already planning to raise the requirement to 3.8 and Jenkins is already at 3.12, so we could easily raise to 3.9 as well to avoid legacy code?  What's the lowest python version we really need to use now that (even debian11 is at 3.9)?

Reply:
moved out